api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -214,6 +214,38 @@ message OktaAssignmentMetadata {
   string EndingStatus = 4 [(gogoproto.jsontag) = "ending_status,omitempty"];
 }
 
+// AccessListMemberMetadata contains common metadata for access list member resource events.
+message AccessListMemberMetadata {
+  // joined_on is the date that the member joined.
+  google.protobuf.Timestamp joined_on = 1 [
```

Comment:
The rest of this file is using `CamelCase` field names. I know snake case is more idiomatic and preferred in our newer protos, but shouldn't we stay consistent here?

Reply:
This was more or less born of a hope to scrape gogo out of this `proto` definition at some point and resisting the urge to create more to cleanup later. I understand about the conssitency, though -- if you'd rather I kept consistency, let me know. But I'd love not to have to mess with this later.